### DISCUSSION TITLE
v2/copy-code/zh/config

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#133] [Announcements]
# v2/copy-code/zh/config

VuePress 的代码一键复制插件

https://vuepress-theme-hope.github.io/v2/copy-code/zh/config.html

**Comment by itlemon** [comment]
你这个默认的selector有点问题，应该是：.theme-default-content div[class*=language-] pre

**Reply by Mister-Hope** [reply]
这跟文档上的有什么不一样的么？如果忽略我两侧用了双引号而不是单引号？

**Reply by itlemon** [reply]
`language`两侧用了双引号，确实没有效果。去掉就好了。

**Reply by itlemon** [reply]
大佬，方便加个微信请教一个问题嘛？itlemon2022

**Reply by Mister-Hope** [reply]
默认的选择器没有问题.... 有问题可以加主题 QQ 群提出
